### ISSUE TITLE
Build ARM Docker images

### ISSUE THREAD

**Issue by akkie** [mockoon/mockoon#670]
**Is your feature request related to a problem? Please describe.**
I want to run the CLI on Apple Silicon devices

**Describe the solution you'd like**
Provide cross builds for AMD64 and ARM

**Comment by gerwazy102** [comment]
I found out today that aarch64 docker images do not exist yet for mockoon. I managed to easily build one locally just by doing `docker build --tag mockoon/cli:latest --platform linux/arm64 .`

I also modified ci to fix this: https://github.com/gerwazy102/mockoon/pull/1/files . I can test this further on my fork if you think this is a step in right direction.

**Comment by 255kb** [comment]
@gerwazy102 Thanks for contributing. I really appreciate, especially as I don't have ARM architecture available 🙂
Could you open a PR with your changes? 
I am wondering, how is this published then on Docker Hub, as two separated images? and how people do select the right architecture when pulling the image? 
The main question being, do we need to update the [CLI docs](https://github.com/mockoon/mockoon/tree/main/packages/cli#docker)?

**Comment by gerwazy102** [comment]
Happy to contribute! Mockoon is a life saver for me! :) I created pull request: https://github.com/mockoon/mockoon/pull/985

When it comes to how multiple architectures are handled by docker - docker supports multi architecture images out of the box and can select correct one based on arch it is running on. We can have same tag "v1.0.0" for amd64 and arm64 without any issues. You can see example for centos on how this works:

```bash
docker manifest inspect centos:8
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 529,
         "digest": "sha256:a1801b843b1bfaf77c501e7a6d3f709401a1e0c83863037fa3aab063a7fdb9dc",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 529,
         "digest": "sha256:65a4aad1156d8a0679537cb78519a17eb7142e05a968b26a5361153006224fdc",
         "platform": {
            "architecture": "arm64",
            "os": "linux",
            "variant": "v8"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 529,
         "digest": "sha256:8bcb67d4c816e16f79bdea8079c5e30e88f404f4aa8fa124245864830d7852d2",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      }
   ]
}

```

**Comment by 255kb** [comment]
This has been released in [v3.0.0](https://mockoon.com/releases/3.0.0/).
